.run/WalletTool CJ Mix.run.xml

```diff
@@ -1,8 +1,8 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="WalletTool CJ Mix" type="Application" factoryName="Application">
     <option name="MAIN_CLASS_NAME" value="org.bitcoinj.tools.WalletTool" />
-    <module name="dashj-master-three.tools.main" />
-    <option name="PROGRAM_PARAMETERS" value="mix --wallet=coinjoin.testnet.wallet --net=TEST --debuglog --amount=0.007 --sessions=4 --rounds=1" />
+    <module name="dashj-feature-coinjoin.tools.main" />
+    <option name="PROGRAM_PARAMETERS" value="mix --wallet=coinjoin.testnet.wallet --net=TEST --debuglog --amount=0.15 --sessions=8 --rounds=1 --multi-session" />
```

Comment:
This is the run config that uses "better" parameters to result in faster mixing

Reply:
Using 8 sessions gave performance improvements over 4 sessions.  It could be that 12 or 16 sessions could be faster.

Additionally, using multisession also was improved the speed.

Neither of these values are hard coded anywhere in DashJ.  Currently WalletTool is still using the default settings of Dash Core, which is 4 sessions and multi-session OFF.  However, these will be hard-coded as constants in DashPay in a future story.